service/src/ServiceProxy.cpp

```diff
@@ -157,6 +157,12 @@ namespace geopm
                                  setting);
     }
 
+    std::string ServiceProxyImp::topo_get_cache(void)
```

Comment:
I see a unit test was added for the changes to PlatformTopo, however, a test should be added for this new method in ServiceProxy.